server/src/test/java/org/elasticsearch/search/retriever/RankDocsRetrieverBuilderTests.java

```diff
@@ -102,32 +97,34 @@ public void testExtractToSearchSourceBuilder() {
         if (randomBoolean()) {
             source.aggregation(new TermsAggregationBuilder("name").field("field"));
         }
+        if (randomBoolean()) {
+            source.explain(true);
+        }
+        if (randomBoolean()) {
+            source.profile(true);
+        }
```

Comment:
```suggestion
        source.explain(randomBoolean());
        source.profile(randomBoolean());
```

I am not sure the reason for the extra indirection :).